test/verify/check-dashboard

```diff
@@ -70,17 +70,17 @@ class TestDashboard(MachineCase):
             b.click('#dashboard_setup_server_dialog .btn-primary')
         b.wait_popdown('dashboard_setup_server_dialog')
 
+class TestBasicDashboard(MachineCase, DashBoardHelpers):
+    def setUp(self):
+        additional_machines = [ { }, { } ]
```

Comment:
How about we make this a dict() so we can refer to machines by name. It'll be clearer and easier to read.